custom/import_xml.php

```diff
@@ -190,24 +193,31 @@ function setInsurance($pid, $ainsurance, $asubscriber, $seq) {
 ?>
 <html>
 <head>
-<?php html_header_show();?>
-<link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
-<title><?php xl('Import Patient Demographics','e'); ?></title>
+    <?php Header::setupHeader(); ?>
+    <title><?php echo xlt('Import Patient Demographics XML'); ?></title>
 </head>
 <body class="body_top" onload="javascript:document.forms[0].form_import_data.focus()">
-
-<p><?php xl('Paste the data to import into the text area below:','e'); ?></p>
-
-<center>
-<form method='post' action="import_xml.php">
-
-<textarea name='form_import_data' rows='10' cols='50' style='width:95%'></textarea>
-
-<p>
-<input type='submit' name='form_import' value=<?php xl('Import Patient','e','\'','\''); ?> /> &nbsp;
-<input type='button' value=<?php xl('Cancel','e','\'','\''); ?> onclick='window.close()' /></p>
+<form method='post' action="import_xml.php" onsubmit="return top.restoreSession()">
+    <div class="container">
+        <div class="row">
+            <div class="col-xs-12">
+                <div class="form-group"></div>
+                <div class="form-group">
+                    <textarea name='form_import_data' class='form-control' rows='10' cols='50'></textarea>
+                </div>
+                <div class="form-group text-right">
+                    <div class="btn-group" role="group">
+                        <button type='submit' class='btn btn-default btn-save' name='form_import' value='bn_import'>
+                            <?php echo xla('Import'); ?>
```

Comment:
I'll check, but I think I caught and fixed this in a subsequent commit.
(btw, that's a good catch and means you are well on your way to being a anal reviewer extraordinaire :))